src/google-maps/map-marker/map-marker.ts

```diff
@@ -34,42 +33,38 @@ export const DEFAULT_MARKER_OPTIONS = {
   selector: 'map-marker',
   exportAs: 'mapMarker',
 })
-export class MapMarker implements OnInit, OnDestroy, MapAnchorPoint {
+export class MapMarker implements OnInit, OnChanges, OnDestroy, MapAnchorPoint {
   private _eventManager = new MapEventManager(this._ngZone);
-  private readonly _options =
-      new BehaviorSubject<google.maps.MarkerOptions>(DEFAULT_MARKER_OPTIONS);
-  private readonly _title = new BehaviorSubject<string|undefined>(undefined);
-  private readonly _position =
-      new BehaviorSubject<google.maps.LatLngLiteral|google.maps.LatLng|undefined>(undefined);
-  private readonly _label =
-      new BehaviorSubject<string|google.maps.MarkerLabel|undefined>(undefined);
-  private readonly _clickable = new BehaviorSubject<boolean|undefined>(undefined);
-  private readonly _destroy = new Subject<void>();
-
-  @Input()
-  set options(options: google.maps.MarkerOptions) {
-    this._options.next(options || DEFAULT_MARKER_OPTIONS);
-  }
 
-  @Input()
-  set title(title: string) {
-    this._title.next(title);
-  }
+  /**
+   * Title of the marker.
+   * See: developers.google.com/maps/documentation/javascript/reference/marker#MarkerOptions.title
+   */
+  @Input() title: string;
```

Comment:
I think we should keep the "setter" aspect of this and have an internal private property, even if we get rid of the BehaviorSubjects. This would keep the API the same, and prevent the issue where a user might get confused about if they should use, for example, the "title" property or the "getTitle" method. The "title" property would show the value as defined by the component, whereas getTitle gets it from the underlying object. They are not necessarily in sync and the underlying object is the ultimate source of truth.

Reply:
The reason why I've been trying to get rid of the setters is that they generate much more code in ES5. E.g. a simple property is basically just `this.foo = 'bar'` inside the constructor, whereas a setter generates the following:

```
Object.defineProperty(Foo.prototype, "foo", {
    get: function () {
        return 'bar';
    },
    enumerable: false,
    configurable: true
});
```

Reply:
That's understandable. You don't think it will be confusing though to have the "title" property as well as the "getTitle" method?

Reply:
The point about it being confusing is totally valid too and there probably isn't a way around it without introducing a bunch of breaking changes.